### PR TITLE
TUP#273 : Set verification of projects and products from submission overview

### PR DESCRIPTION
**Pull request by rafinala**

### CODE REVIEW

File: controllers/ChallengeController.php [hunk 1]
```diff
@@ -174,4 +175,24 @@ public function actionEdit()
             ]
         );
     }
+    public function actionReviewFunding($id, $status)
+    {
+        $model = Funding::findOne(['id' => $id]);
+
```
Comment (gdalyy) [inline controllers/ChallengeController.php:181]: missign a 404 NOT_FOUND handling , what if the giving  `id` doesn't match any funding ?

File: controllers/MarketplaceController.php [hunk 2]
```diff
@@ -143,4 +145,25 @@ public function actionEdit()
             ]
         );
     }
+    public function actionReviewProduct($id, $status)
+    {
+
+        $model = Product::findOne(['id' => $id]);
+
```
Comment (gdalyy) [inline controllers/MarketplaceController.php:152]: missing 404 NOT_FOUND handling here also

File: views/challenge/overview.php [hunk 3]
```diff
@@ -93,112 +95,122 @@
                         $space = $funding->getSpace()->one();
                         $cover = $funding->getCover();
                         ?>
-                        <a href="<?= $space->createUrl('/xcoin/funding/details', [
-                            'container' => $this->context->contentContainer,
-                            'fundingId' => $funding->id
-                        ]); ?>"
-                            data-target="#globalModal">
+                        <div style="position: relative">
```
Comment (gdalyy) [inline views/challenge/overview.php:98]: @rafinala why we have a `style="position: relative"` here , can you ask mortadha how to change this in a css class ?

File: views/marketplace/overview.php [hunk 4]
```diff
@@ -133,107 +135,115 @@
                         $owner = $product->isSpaceProduct() ? $product->getSpace()->one() : $product->getCreatedBy()->one();
                         $picture = $product->getPicture();
                         ?>
-                        <a href="<?= $owner->createUrl('/xcoin/product/details', [
-                            'productId' => $product->id
-                        ]); ?>"
-                            data-target="#globalModal">
+                        <div style="position: relative">
```
Comment (gdalyy) [inline views/marketplace/overview.php:138]: same here

**Review by gdalyy — DISMISSED** [review]

File: views/challenge/overview.php (hunk 3 above)
Comment (rafinala) [reply views/challenge/overview.php:98]: yes morthadha will work on the style for this and market place overview

**Review by gdalyy — APPROVED** [review]

**Pull request merged**